compiler/rustc_hir_analysis/src/check/errs.rs

```diff
@@ -66,32 +66,24 @@ fn handle_static_mut_ref(
     hir_id: hir::HirId,
 ) {
     if e2024 {
-        let sugg = if mutable {
-            errors::StaticMutRefSugg::Mut { span, var }
+        let (sugg, shared) = if mutable {
+            (errors::StaticMutRefSugg::Mut { span, var }, "mutable ")
         } else {
-            errors::StaticMutRefSugg::Shared { span, var }
+            (errors::StaticMutRefSugg::Shared { span, var }, "shared ")
```

Comment:
Why do you put the space here, rather than adding it in the ftl file?

Reply:
The suggestion immediately above this one answers your question - in order to not create a double space. But it's wrong because the ftl file needs structure changes for EVERY other language since they can't embed the variable directly

Reply:
All values for `shared` end in a space. So we can just remove the space here and add it in the ftl file without changing output I think?